query-connector/src/app/constants.ts

```diff
@@ -300,7 +300,7 @@ export const stateOptions = [
 export type Mode = "search" | "results" | "select-query" | "patient-results";
 
 /* Mode that query building pages can be in; determines what is displayed to the user */
-export type BuildStep = "condition" | "valueset" | "concept";
+export type BuildStep = "selection" | "condition" | "valueset" | "concept";
```

Comment:
makes sense. I don't think we actually need a `concept` BuildStep, since concepts are selected as part of the Drawer component on the valueset page.